### PR TITLE
Update UG and UI related issues.

### PR DESCRIPTION
**Pull request by Morrow1ndy**

### CODE REVIEW

File: src/main/java/seedu/address/ui/MemberDetailsCard.java [hunk 1]
```diff
@@ -101,7 +99,6 @@ public boolean equals(Object other) {
 
         // state check
         MemberDetailsCard card = (MemberDetailsCard) other;
-        return index.getText().equals(card.index.getText())
-                && member.equals(card.member);
+        return member.equals(card.member);
     }
```
Comment (zzybluebell) [inline src/main/java/seedu/address/ui/MemberDetailsCard.java:103]: Great

File: docs/UserGuide.md [hunk 2]
```diff
@@ -428,7 +449,7 @@ If your changes to the data file makes its format invalid, ezFoodie will discard
 **A**: Install the application in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous ezFoodie home folder.
 
 **Q**: Why my ezFoodie does not have any populated sample member data?<br>
-**A**: Delete `ezFoodie.json` and restart the application. Refer to #22 in **Features** above to find this JSON file.
+**A**: Delete `ezFoodie.json` and restart the application. Refer to #22 in *Features* above to find this JSON file.
```
Comment (zzybluebell) [inline docs/UserGuide.md:453]: Good

File: docs/UserGuide.md [hunk 3]
```diff
@@ -145,21 +153,21 @@ Continue reading to explore the wonders of ezFoodie and enhance the way you mana
...
 
-Exits the program.
+Exit the program.
 
 Format: `exit`
 
 ### 3. Logging in as a manager : `login`
 
-Logs in as a manager.
+Log in as a manager.
```
Comment (zzybluebell) [inline docs/UserGuide.md:171]: I think should add "s" back according to original AB3 UG format

Comment (Morrow1ndy) [reply docs/UserGuide.md]: Will do.

**Review by HolmesJJ — APPROVED** [review]
LGTM

**Pull request merged**